securing_apps/topics/oidc/java/jboss-adapter.adoc

```diff
@@ -102,6 +102,8 @@ This ZIP archive contains JBoss Modules specific to the {project_name} adapter.
 
 To configure the adapter subsystem if the server is not running execute:
 
+NOTE: Alternatively, you can specify `server.config` property when installing adapters in command line to install adapters into different config. E.g. `-Dserver.config=standalone-ha.xml`
+
```

Comment:
NOTE: Alternatively, you can specify the `server.config` property while installing adapters from the command line to install adapters using a different config, for example: `-Dserver.config=standalone-ha.xml`.